docs/install/any-kubernetes-cluster.md

```diff
@@ -605,8 +605,25 @@ kubectl apply --filename {{< artifact repo="eventing" file="in-memory-channel.ya
 1. Install a Broker (eventing) layer:
 
       <!-- This indentation is important for things to render properly. -->
-
+      
    {{< tabs name="eventing_brokers" default="MT-Channel-based" >}}
+   
+   {{% tab name="Apache Kafka Broker" %}}
+   {{< feature-state version="v0.17" state="alpha" >}}
+   
+The following command installs an implementation of Broker that utilizes
+Apache Kafka natively and runs event routing in a system namespace, 
+`knative-eventing` by default.
+
+<!-- TODO Where does the config for populating the following live? -->
```

Comment:
Does this command actually work? Not sure what this TODO means.

Reply:
Yes, it does, I forgot to remove it.